app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -414,8 +413,7 @@ fun PoweredByPocketHeader(
                 Text(
                     text = stringResource(R.string.pocket_stories_feature_title),
                     color = textColor,
-                    fontSize = 12.sp,
-                    lineHeight = 16.sp
+                    style = FirefoxTheme.typography.caption
```

Comment:
```suggestion
                    style = FirefoxTheme.typography.caption,
```
I think this is fine, but I am not too sure about it since the designs don't technically specify `caption` text style here, and are a bit of outdate since we can see it using the Inter font. You can proceed with landing it as is or double check with UX that this bit of text should also be `caption`.

Reply:
Asked for UX feedback.

Reply:
Crystal confirmed `caption` is right.